mne_bids/mne_bids.py

```diff
@@ -67,6 +69,11 @@ def _channels_tsv(raw, fname, verbose):
                     eog='ElectrOculoGram', misc='Miscellaneous',
                     ref_meg='Reference channel')
 
+    ignored_indexes = []
+    for ch_name in IGNORED_CHANNELS:
```

Comment:
you shouldn't ignore this by default. It should be ignored only if it's synthetic. Otherwise, it should be kept in

Reply:
is it ever *not* synthetic?
Also, how would you propose determining whether it is synthetic or not?

Reply:
in `fif` systems, it corresponds to the stim channel. It's not synthesized by mne-python. We should just look at the extension and determine if it's synthetic or not (based on how MNE reads the different file formats)

Reply:
FWIW, these synthetic channels will soon be moved into `raw.annotations` but at the moment it's still an open PR by @agramfort